shared/common-adapters/user-bio.desktop.js

```diff
@@ -59,7 +61,10 @@ export default class BioRender extends Component<void, Props, void> {
 
     return (
       <Box style={{minHeight: 170, ...this.props.style}}>
-        <BioLoading loading={loading} style={this.props.style} avatarSize={this.props.avatarSize} />
+
+        <ReactCSSTransitionGroup transitionName='no-anim' transitionEnterTimeout={250} transitionLeaveTimeout={250}>
+          {loading && <BioLoading key='loading-state' loading={loading} style={this.props.style} avatarSize={this.props.avatarSize} />}
```

Comment:
The bioloading is a little tricky, because the avatar will only fade in/out if you change the opacity of it directly, otherwise it just pops in/out.

So the simplest thing was to keep the existing animation effects and just use `ReactCSSTransistionGroup` to remove the loading component when we don't need it anymore. So the transitionName `no-anim` means this group isn't getting animated by `ReactCSSTransistionGroup` (open to better names).